app/models/permitted_queries.rb

```diff
@@ -40,7 +40,7 @@ def permitted_apos
         fq: ['objectType_ssim:adminPolicy', '!project_tag_ssim:Hydrus']
       )
       .dig('response', 'docs')
-      .sort_by { |doc| doc.fetch(SolrDocument::FIELD_TITLE).first.downcase.delete('[]') }
+      .sort_by { |doc| doc.fetch(SolrDocument::FIELD_TITLE).downcase.delete('[]') }
```

Comment:
the new display field is now single valued because ... that's how we use it - as a display field only, single valued.